Makefile

```diff
@@ -2,18 +2,16 @@
 
 DESTDIR=build
 
-all: ercole-agent
-
-default: ercole-agent
+all: linux windows
 
 clean:
-	rm -rf ercole-agent build ercole-agent.exe *.exe
+	rm -rf ercole-agent build ercole-agent.exe *.exe main
 
-ercole-agent:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ercole-agent -a -x
+linux:
+	GOOS=linux GOARCH=amd64 GO111MODULE=on CGO_ENABLED=0 go build -o ercole-agent -tags linux -a
```

Comment:
Why you need to set linux as build tag? go build seems to auto set linux build tag if GOOS=linux
https://golang.org/pkg/go/build/

Reply:
I didn't know!
Fixed :+1: